addons/packages/grafana/7.5.11/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: grafana/grafana:7.5.11
+  image: projects.registry.vmware.com/tkg/grafana/grafana@sha256:df8f25cc9ee43d6ea4c22f9c6c46644e2b9a485562dd0dafe831b5b582ac0a71
```

Comment:
@seemiller I want to change this line to let tce use Grafana upstream image, do you know how to update this line?

Reply:
that file is created by kbld.

Reply:
You'd have to delete the `kbld-config.yaml` file and then run
`kbld --file bundle --imgpkg-lock-output bundle/.imgpkg/images.yml` from the `addons/packages/grafana/7.5.11` directory.
Then push up to Harbor and update the `package.yaml` with the new sha.

Reply:
However, I think that we're using `projects.registry.vmware.com` because of rate limiting issues with Docker

Reply:
We can use `imgpkg copy` to copy the `grafana/grafana:7.5.11` from Docker to the TCE Harbor project, assuming that you don't want to use the TKG Harbor project.

Reply:
@seemiller yes, we want something like fluent-bit, not sure how to get the sha?
https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/fluent-bit/1.7.5/bundle/.imgpkg/images.yml#L6

Reply:
@seemiller I've discussed within our team and decided to use upstream image here